.travis.yml

```diff
@@ -1,5 +1,13 @@
 language: c
 
+env:
+  - TESTS=
+  - TESTS=--enable-tests TESTLEVEL=unit
+  - TESTS=--enable-tests TESTLEVEL=basic
+  - TESTS=--enable-tests TESTLEVEL=extra1
+  - TESTS=--enable-tests TESTLEVEL=extra2
```

Comment:
This setup would not work correctly as now matrix is defined explicitly below. Better would be run all tests (unit, basic, extra1, extra2) in one step and ideally for all builds which use unversioned compiler version. You can (re)use CONFIGURE_FLAGS env to set "--enable-tests".

Reply:
Well, I tried run all tests in one job, but it exceeds maximum run time (downloading test images is slow). I think there are three possible solutions:
1. Faster test archive hosting
2. Splitting testing into separate jobs
3. Caching in travis (not sure about that, because if all three image sets will be present at same time in job, it will run out of space. Therefore it would be possible only in case that cache won't be counted as used space)

Reply:
What about putting these test images into separate github repository and download them via git clone --depth=1? Would not it be faster?

Anyway, running tests should be enough for builds with compilers in build matrix which do not have specified version (currently build jobs 35-55). This can be achieved e.g. by adding CONFIGURE_FLAGS="--enable-tests" into those env jobs definitions.

Reply:
It was originally in separate repository, but GitHub was complaining about its size. That is true reason why it is hosted on private server. 

Well, job died because of exceeding maximum time. So running it for selected jobs wouldn't solve that problem.

Reply:
I tried in meanwhile git-lfs, but there is limit for storage (1 GB) and traffic (1 GB/mo). 

A this moment I am using releases as you recommended. It seems it works because only actual limitation should be single file size (max 2 GB).

If it will be ok, I'll keep it like that.